### PR TITLE
[refactor] use Aarch64VCpuCreateConfig as AxVCpuCreateConfig in aarch64

### PR DESCRIPTION
**Pull request by hky1999**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -23,7 +23,7 @@ page_table_multiarch = "0.4.1"
 percpu = { version = "0.1.4", features = ["arm-el2"] }
 
 # System dependent modules provided by ArceOS-Hypervisor.
-axvcpu = { git = "https://github.com/arceos-hypervisor/axvcpu.git" }
+axvcpu = { git = "https://github.com/arceos-hypervisor/axvcpu.git", branch = "smp" }
```
Comment (aarkegz) [inline Cargo.toml:26]: Maybe remove this `branch` option？

File: Cargo.toml [hunk 2]
```diff
@@ -35,4 +35,10 @@ x86_vcpu = { git = "https://github.com/arceos-hypervisor/x86_vcpu.git" }
 riscv_vcpu = { git = "https://github.com/arceos-hypervisor/riscv_vcpu.git" }
 
 [target.'cfg(target_arch = "aarch64")'.dependencies]
-arm_vcpu = { git = "https://github.com/arceos-hypervisor/arm_vcpu.git" }
+arm_vcpu = { git = "https://github.com/arceos-hypervisor/arm_vcpu.git", branch = "smp" }
```
Comment (aarkegz) [inline Cargo.toml:38]: And this？ After we merge this.

**Review by aarkegz — DISMISSED** [review]

File: Cargo.toml (hunk 1 above)
Comment (hky1999) [reply Cargo.toml]: updating

**Review by aarkegz — APPROVED** [review]
LGTM

**Pull request merged**